src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py

```diff
@@ -172,7 +172,7 @@ def __call__(
             self.unet.in_channels,
         )
         if latents is None:
-            latents = jax.random.normal(prng_seed, shape=latents_shape, dtype=self.dtype)
+            latents = jax.random.normal(prng_seed, shape=latents_shape, dtype=jnp.float32)
         else:
             if latents.shape != latents_shape:
```

Comment:
This file doesn't seem to be in `main`. Should be removed from here before mergin

Reply:
The merge target is `flax_pipeline` for now.